crates/astria-eyre/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "astria-eyre"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+eyre = { workspace = true }
```

Comment:
do any other crates depend on `eyre` now? should this be non-workspace?

Reply:
Updated. `eyre` is no longer a workspace dependency.